scan/unstaged.go

```diff
@@ -268,12 +268,12 @@ func diffPrettyText(diffs []diffmatchpatch.Diff) string {
 
 // gitStatus returns the status of modified files in the worktree. It will attempt to execute 'git status'
 // and will fall back to git.Worktree.Status() if that fails.
-func gitStatus(wt *git.Worktree) (git.Status, error) {
+func gitStatus(wt *git.Worktree, staggedOnly bool) (git.Status, error) {
 	c := exec.Command("git", "status", "--porcelain", "-z")
 	c.Dir = wt.Filesystem.Root()
 	output, err := c.Output()
 	if err != nil {
-		stat, err := wt.Status()
+		stat, _ := wt.Status()
```

Comment:
@thewizzy wouldn't you want that to stay `err` there? That way the code using it can continue on with `wt.Status()` as a fallback?

It looks like if it returned the err from `c.Output()` then some code would find that `err != nil` and wouldn't continue processing.

Reply:
Good catch. Thank you.